deployment/helm-chart/README.md

```
@@ -0,0 +1,122 @@
+Note: Without authentication, once you publish this on the internet, anyone can access your app, see your data, and modify your settings!
+
+## Dependencies used in this kustomization:
+- nginx.ingress
+- external-dns
+- cert-manager
+- kubectl
+- helm 
+
+## Steps to deploy:
+
+`Suggestion`: Before deploying, check out the Helm values in the `deployment/helm ` file and modify them to suit your Kubernetes environment.
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)